browser/src/slideshow/PresenterConsole.js

```diff
@@ -266,6 +316,14 @@ class PresenterConsole {
 		}
 	}
 
+	_onTransitionStart(e) {
+		if (!this._proxyPresenter) {
+			return;
+		}
+
+		this._currentIndex = e.slide;
+	}
```

Comment:
So, now we're not fetching at all, here. My hint was just to avoid to display the preview in advance.
Anyway, if you think that pre-fetching every 2 sec is enough and we don't need anymore to do it here, this method could be completely removed since it does nothing.
On the contrary if you think that it could be helpful to fetch next slide here, you could extract the fetching part from `drawNext` so that we can just fecth the next slide preview without showing it.
Please, do whatever you think it's more appropriate :-)

Reply:
Oops, sorry I forgot to include the stage code. Fixed it.

Reply:
Okay, I see.
Maybe it would be better to pass the index value to `_fetchPreviews` without modifying `this._lastIndex` value

Reply:
It is a good idea, thanks